src/main/java/frc/team2412/robot/subsystem/DrivebaseSubsystem.java

```diff
@@ -256,30 +285,33 @@ public double getAngularVelocity() {
 
     public Rotation2 getAngle() {
         synchronized (kinematicsLock) {
-            return getPose().rotation;
-            // return Robot.getInstance().isCompetition() ? getPose().rotation.inverse() : getPose().rotation;
+            return Robot.getTypeFromAddress() == Robot.RobotType.DRIVEBASE ? getPose().rotation.inverse()
+                    : getPose().rotation;
```

Comment:
This will be slow - please use a cached value from Robot that determines drivebase vs. competition robot

Reply:
Switched to using isCompetition